invenio_oauthclient/contrib/keycloak/handlers.py

```diff
@@ -45,7 +45,7 @@
 from .helpers import get_user_info
 
 
-def info_serializer_handler(remote, resp, user_info, **kwargs):
+def info_serializer_handler(remote, resp, token_user_info, user_info, **kwargs):
```

Comment:
Very minor:
```suggestion
def info_serializer_handler(remote, resp, token_user_info=None, user_info=None, **kwargs):
```
It looks like, from the code, that they could be both `None` (in case of errors and/or the remote fetch is disabled).
If that's the case, I think this should be reflected in the code and implementation of this method.

Ideally, I would have `token_user_info` always set, with default `{}`, and `user_info` to be `{}` or `None`... maybe `None` would be better when disabled, it means that it was not called (while `{}` means that it was called, but no values returned).